lib/task_bunny/config.ex

```diff
@@ -59,7 +66,7 @@ defmodule TaskBunny.Config do
   end
 
   @doc """
-  Returns workers in config.
+  Transforms queue configuration into list of workers application should run.
```

Comment:
Transforms queue configuration into a list of workers, which the application should run.